src/index.html

```diff
@@ -15,8 +15,96 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav>
+        <ul class="menu-list">
+          <li class="list-item">
+            <a
+              href="/"
+              class="item-link is-active"
+            >
+              apple
```

Comment:
The link text 'apple' should be styled to uppercase using CSS, rather than being hardcoded in lowercase in the HTML. This applies to all link texts in the menu.